public/API/API_GetConsoleIDs.php

```diff
@@ -9,6 +9,20 @@
  *   object    [value]
  *    string    ID                  unique identifier of the console
  *    string    Name                name of the console
+ *    string    IconURL             System icon URL
```

Comment:
```suggestion
 *    string    IconURL             system icon URL
```

nit: Just to remain consistent with the other API doc comments.